index.bs

```diff
@@ -176,19 +185,101 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
 
 In general, it is recommended to use properties instead of [=Metadata OBUs=] in the [=AV1 Item Configuration Property=].
 
-For multi-layered images, the LayerSelectorProperty may be present as defined in [[!HEIF]].
+<h3 id ="layered-items">AV1 Layered Image Items</h3>
+
+<h4 id ="layered-items-overview">Overview</h4>
+
+[[!AV1]] supports encoding a frame using multiple spatial layers. A spatial layer may improve the resolution or quality of the image decoded based on one or more of the previous layers. A layer may also provide an image that does not depend on the previous layers. Additionally, not all layers are expected to produce an image meant to be rendered. Some decoded images may be used only as intermediate decodes. Finally, layers are grouped into one or more [=Operating Points=]. The [=Sequence Header OBU=] defines the list of [=Operating Points=], provides required decoding capabilities, and indicates which layers form each [=Operating Point=].
+
+[[!AV1]] delegates the selection of which [=Operating Point=] to process to the application. AVIF defines the [=OperatingPointSelectorProperty=] to control this selection. In the absence of an [=OperatingPointSelectorProperty=] associated with an [=AV1 Image Item=], the AVIF renderer is free to process any [=Operating Point=] present in the [=AV1 Image Item Data=]. In particular, when the [=AV1 Image Item=] is composed of a unique [=Operating Point=], the [=OperatingPointSelectorProperty=] should not be present. If an [=OperatingPointSelectorProperty=] is associated with an [=AV1 Image Item=], the <code>op_index</code> field indicates which [=Operating Point=] is expected to be processed for this item.
+
+NOTE: When an author wants to offer the ability to render multiple [=Operating Points=] from the same AV1 image (e.g. in the case of multi-view images), multiple [=AV1 Image Items=] can be created that share the same [=AV1 Image Item Data=] but have different [=OperatingPointSelectorProperty=]s.
+
+[[!AV1]] expects the renderer to display only one frame within the selected [=Operating Point=], which should be the highest spatial layer that is both within the [=Operating Point=] and present within the temporal unit, but [[!AV1]] leaves the option for other applications to set their own policy about which frames are output. AVIF sets a different policy, and defines how the 'lsel' property (defined in [[!HEIF]]) is used to control which layer is rendered. In the absence of a 'lsel' property associated with an [=AV1 Image Item=], the renderer is free to render either: only the output image of the highest spatial layer, or to render all output images of all the intermediate layers, resulting in a form of progressive decoding. If a 'lsel' property is associated with an [=AV1 Image Item=], the renderer is expected to render only the output image for that layer.
```

Comment:
The sentence "In the absence of a 'lsel' property associated with an [=AV1 Image Item=], the renderer is free to render ..." implies that an AV1 Image Item with multiple layers is allowed to have no 'lsel' property associated with it. This seems to contradict the following requirement in HEIF Section 6.5.11.1:

```
If the decoding of a multi-layer image item results into more than one
reconstructed image, the 'lsel' item property shall be associated with
the image item. ...
```

Note: The paragraph "In the absence of a 'lsel' property associated with the item:" at line 231 also implies that an AV1 image item with multiple layers may have no 'lsel' property associated with it.

Reply:
Good point. My interpretation of the HEIF spec is that the use of `lsel` is codec-specific. We could ask to have that sentence clarified in HEIF.

Reply:
I opened an issue on HEIF: https://github.com/MPEGGroup/FileFormat/issues/24